DCC.schema.json

```diff
@@ -3,32 +3,53 @@
   "$id": "https://id.uvci.eu/DCC.schema.json",
   "title": "EU DCC",
   "description": "EU Digital Covid Certificate",
-  "$comment": "Schema version 1.2.1",
-  "required": [
-    "ver",
-    "nam",
-    "dob"
-  ],
+  "$comment": "Schema version 1.3.0",
   "type": "object",
+  "oneOf": [
```

Comment:
Just as @martin-lindstrom noted, this is a breaking change. On the other hand, there are several of them in this revision.